Structure_Engine/Query/SelectCase.cs

```diff
@@ -56,7 +65,7 @@ public static partial class Query
                 else if (loadcase is int || loadcase is double)
                     loadCaseId = loadcase.ToString();
 
-                if (!string.IsNullOrWhiteSpace(loadCaseId))
+                if (string.IsNullOrWhiteSpace(loadCaseId))
```

Comment:
Pretty sure this was right as it was
```suggestion
                if (!string.IsNullOrWhiteSpace(loadCaseId))
```